### PR TITLE
Added remote chat

### PR DESCRIPTION
**Pull request by meltingice1337**
The chat is hidden from taskmanager and it will allways stay on top issue #351

### CODE REVIEW

File: Client/Forms/FrmRemoteChat.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        }
+
+        private void StartKeepingTopMost()
+        {
+            (new Thread(() => 
+            {
+                while (Visible)
+                {
+                    NativeMethods.SetForegroundWindow(Handle);
+                    Thread.Sleep(1);
```
Comment (yankejustin) [inline Client/Forms/FrmRemoteChat.cs]: This seems a bit too low. Sleeping the thread only every 1 millisecond certainly would decrease performance pretty badly. Perhaps sleep for 20 or even 50 seconds?

**Comment by meltingice1337** [comment]
Here is an image of the chat http://i.imgur.com/wkPp5f9.png

File: Client/Forms/FrmRemoteChat.cs (hunk 1 above)
Comment (meltingice1337) [inline Client/Forms/FrmRemoteChat.cs]: Yes, or maybe, it will only be set on the foreground when it receives a message ??

**Comment by MaxXor** [comment]
I've not forgot about this, but I won't add this now before I've released v1.3.0.0. Thanks for your contribution!

**Comment by DragonzMaster** [comment]
Can we use Chat bubbles like that :smile: !?
http://stackoverflow.com/questions/30764357/control-snapping-on-resize-in-c-sharp-scrollable-container
I think it would be nice :)

**Comment by BlackEyes250** [comment]
Hi @MaxXor when will be relased v1.3.0.0 ?

**Comment by MaxXor** [comment]
@BlackEyes250 I try to release it this week, only one issue left to get fixed. #410

**Comment by LostSoulFly** [comment]
Still hoping this gets added to the main branch!

**Comment by dmgamingstudios** [comment]
Weird thing, when I tried to merge your chat with my local source copy the entire application broke, everything i want to open, wether its system information or keylogger logs, the client disconnects with unexpectedendofstream exception, trying to remove the chat again didnt fix it and the client couldnt connect anymore as it disposed the socket due to the previous exception

**Comment by Gecco12** [comment]
Will this feature be added in this version? Still isn't added? o_O

**Comment by Knakiri** [comment]
after closing the frmchat clients chat doesn't go away maybe i did something wrong?

**Comment by ErickDaCoder** [comment]
> after closing the frmchat clients chat doesn't go away maybe i did something wrong?

this is probably by design so the client has a copy of the exchange. then again it would be useful to be able to close it manually from the administration panel.

**Pull request closed without merging**